scripts/mkcloud

```diff
@@ -1235,6 +1245,16 @@ function sanity_checks()
     if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
+
+    if [[ " ${steplist[*]} " == *" batch "* ]] ; then
+        need_scenario=1
+        if [[ -n $scenario ]] ; then
+            scenariofile="${mkcloud_dir}/scenarios/cloud$(getcloudver)/${scenario}.yaml"
```

Comment:
I'm not really sure.  I agree they're not scripts but OTOH they are directly related to mkcloud ... I guess it doesn't matter much.  TBH the directory layout is already a mess :-/